File: JavaScript_Courses/JavaScript_Basics/Preparation/04_For_Loop_Lab/06_Vowels_Sum.js
function vowelsSum(input) {
    let text = input[0];

    let sum = 0;
    for (let i = 0; i < text.length; i++) {
        let currentChar = text[i];
        if (currentChar == "a") {
            sum += 1;
        } else if (currentChar == "e") {
            sum += 2;
        } else if (currentChar == "i") {
            sum += 3;
        } else if (currentChar == "o") {
            sum += 4;
        } else if (currentChar == "u") {
            sum += 5;
        }
    }

    console.log(sum);
}

// vowelsSum(["bamboo"]);

File: Python_Courses/Python_Basics/Preparation/04_For_Loop_Lab/06_Vowels_Sum.py
# text = input()
# total_sum = 0
#
# for i in range(len(text)):
#     char = text[i]
#     if char == "a":
#         total_sum += 1
#     elif char == "e":
#         total_sum += 2
#     elif char == "i":
#         total_sum += 3
#     elif char == "o":
#         total_sum += 4
#     elif char == "u":
#         total_sum += 5
#
# print(total_sum)

char_values = {
    "a": 1,
    "e": 2,
    "i": 3,
    "o": 4,
    "u": 5,
}

text = input()
total_sum = 0

for i in range(len(text)):
    char = text[i]
    if char in char_values:
        total_sum += char_values[char]

print(total_sum)
